### PR TITLE
Init viction chain

### PR DESCRIPTION
**Pull request by Hosuke**
Init viction chain

### CODE REVIEW

File: dbt_subprojects/tokens/models/tokens/viction/_schema.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - name: tokens_viction_erc20
+    meta:
+      blockchain: viction
+      sector: tokens
+      contributors: hosuke
+    config:
+      tags: ['tokens', 'viction', 'erc20']
+    description: >
+        Viction ERC20 tokens
+    tests:
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/tokens/viction/_schema.yml:13]:
```suggestion
    data_tests:
```

File: dbt_subprojects/tokens/models/prices/viction/prices_viction_tokens.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+SELECT
+    token_id
+    , '{{ blockchain }}' as blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('tomo-tomochain', 'WVIC', 0xC054751BdBD24Ae713BA3Dc9Bd9434aBe2abc1ce, 18)
```
Comment (Hosuke) [inline dbt_subprojects/tokens/models/prices/viction/prices_viction_tokens.sql:21]: FYI, `tomochain` is now `viction`:
https://api.coinpaprika.com/v1/coins/tomo-tomochain

Comment (jeff-dude) [reply dbt_subprojects/tokens/models/prices/viction/prices_viction_tokens.sql:21]: good callout, relevant blog post:
https://blog.viction.xyz/introducing-viction/

**Review by jeff-dude — APPROVED** [review]
thank you!

**Pull request merged**